### PR TITLE
Remove owner

### PR DESCRIPTION
**Pull request by thogard785**

### CODE REVIEW

File: contracts/auction-handler/FastLaneAuctionHandler.sol [hunk 1]
```diff
@@ -94,24 +89,6 @@ contract FastLaneAuctionHandler is FastLaneAuctionHandlerEvents, Ownable, Reentr
...
-
-    constructor(uint24 _initialStakeShare, uint256 _minRelayBidAmount) {
-        flStakeShareRatio = _initialStakeShare;
-        minRelayBidAmount = _minRelayBidAmount;
-        emit RelayInitialized(_initialStakeShare, _minRelayBidAmount);
-    }
-
 
     /// @notice Submits a flash bid
     /// @dev Will revert if: already won, minimum bid not respected, not from EOA, or current validator is not participating in PFL.
```
Comment (BenSparksCode) [inline contracts/auction-handler/FastLaneAuctionHandler.sol:94]:
```suggestion
    /// @dev Will revert if: already won, minimum bid not respected, or not from EOA
```
Looks like there's no longer any check to see if validators are participating in PFL

File: contracts/auction-handler/FastLaneAuctionHandler.sol [hunk 2]
```diff
@@ -162,10 +151,10 @@ contract FastLaneAuctionHandler is FastLaneAuctionHandlerEvents, Ownable, Reentr
         bytes32 _oppTxHash, // Target TX
         address _searcherToAddress,
         bytes calldata _searcherCallData 
-        ) external payable nonReentrant whenNotPaused onlyEOA {
+        ) external payable nonReentrant onlyEOA {
 
             // Relax check on min bid amount for simulated
-            if (_searcherToAddress == address(0) || bid_simulator_enabled == false /* || _bidAmount < minRelayBidAmount */) revert RelaySearcherWrongParams();
+            if (_searcherToAddress == address(0) /* || _bidAmount < minRelayBidAmount */) revert RelaySearcherWrongParams();
```
Comment (BenSparksCode) [inline contracts/auction-handler/FastLaneAuctionHandler.sol:157]:
```suggestion
            if (_searcherToAddress == address(0) revert RelaySearcherWrongParams();
```
minRelayBidAmount has been removed so no need for this commented part anymore

File: contracts/auction-handler/FastLaneAuctionHandler.sol [hunk 3]
```diff
@@ -519,18 +416,29 @@ contract FastLaneAuctionHandler is FastLaneAuctionHandlerEvents, Ownable, Reentr
...
+    modifier onlyCurrentValidator() {
+        require(msg.sender != address(0), "msg.sender = 0 address");
+        require(msg.sender == block.coinbase || msg.sender == validatorsDataMap[block.coinbase].payee, "only this block's validator");
         _;
     }
 
-    modifier onlyEOA() {
-        if (msg.sender != tx.origin) revert RelayPermissionSenderNotOrigin();
+    modifier onlyActiveValidators() {
+        require(msg.sender != address(0), "msg.sender = 0 address");
```
Comment (BenSparksCode) [inline contracts/auction-handler/FastLaneAuctionHandler.sol:426]:
```suggestion
```
This should be effectively impossible (implies someone found the keys to the zero address), I don't think we need to check for this

File: contracts/auction-handler/FastLaneAuctionHandler.sol [hunk 4]
```diff
@@ -519,18 +416,29 @@ contract FastLaneAuctionHandler is FastLaneAuctionHandlerEvents, Ownable, Reentr
     |             Modifiers             |
     |__________________________________*/
 
-    modifier whenNotPaused() {
-        if (paused) revert RelayPermissionPaused();
+    modifier onlyCurrentValidator() {
+        require(msg.sender != address(0), "msg.sender = 0 address");
```
Comment (BenSparksCode) [inline contracts/auction-handler/FastLaneAuctionHandler.sol:420]:
```suggestion
```
See comment below/above, effectively impossible, no need to check

File: contracts/auction-handler/FastLaneAuctionHandler.sol (hunk 3 above)
Comment (thogard785) [reply contracts/auction-handler/FastLaneAuctionHandler.sol:426]: @Elyx0 thoughts?

Comment (Elyx0) [reply contracts/auction-handler/FastLaneAuctionHandler.sol:426]: Ben comment is true, msg.sender will never be address(0)

File: contracts/auction-handler/FastLaneAuctionHandler.sol (hunk 4 above)
Comment (Elyx0) [reply contracts/auction-handler/FastLaneAuctionHandler.sol:420]: Ben comment is true, msg.sender will never be address(0)

**Pull request merged**